### ISSUE TITLE
Eastimator 中增加 loggingTensorHook 以便在日志中输出 loss

### ISSUE THREAD

**Issue by workingloong** [shenweichen/DeepCTR#401]
**Is your feature request related to a problem? Please describe.**
使用 estimator 训练的时候日志中看不到 loss

**Describe the solution you'd like**

```python
       losses = get_losses()
        loss = losses.compute_weighted_loss(
            unweighted_loss, weights=1.0, reduction=losses.Reduction.SUM)
        reg_loss = losses.get_regularization_loss()

        training_loss = loss + reg_loss
        logging_hook = tf.train.LoggingTensorHook(
            {"loss": training_loss}, every_n_iter=10
        )
        training_hooks=[logging_hook]

        eval_metric_ops = self._eval_metric_ops(labels, logits, pred, unweighted_loss)
        return tf.estimator.EstimatorSpec(
            mode=mode,
            predictions=predictions,
            loss=training_loss,
            train_op=train_op_fn(training_loss),
            eval_metric_ops=eval_metric_ops,
           training_hooks=training_hooks,
            training_chief_hooks=training_chief_hooks)
```

**Comment by workingloong** [comment]
I have mistaken it and close it
